Yoast/yoast-components
# Improve content analysis alignments.

## Summary

This PR can be summarized in the following changelog entry:

* Improved alignments of icons and text in the Content Analysis.

## Test instructions

- In the plugin, yarn link the yoast-components or set this branch in package.json
- build the assets
- check alignments on the content analysis and readability analysis in all the metaboxes (posts, terms, attachments)

Screenshot of the improved alignments:

![screen shot 2017-11-24 at 15 15 26](https://user-images.githubusercontent.com/1682452/33214057-96963338-d12a-11e7-83e2-aadc4118fefa.png)

Note for designers: aligning icons of different size and shapes is difficult, almost impossible. especially when they have even / odd widths. An icon that's 16px wide can't be horizontally aligned with one that's 13px wide (theoretically yes, relying on browsers sub-pixel roundings and crazy unit measures but it would be better to have icons that fit nicely with each other) /cc @hedgefield 
For this reason, I've decided to keep the icons aligned on the left:

<img width="1012" alt="screen shot 2017-11-24 at 15 04 36" src="https://user-images.githubusercontent.com/1682452/33214195-247c4ffc-d12b-11e7-8244-52387a448016.png">


Fixes https://github.com/Yoast/wordpress-seo/issues/8315
